Jenkinsfile

```diff
@@ -5,53 +5,51 @@ pipeline {
 
     options {
         buildDiscarder(logRotator(numToKeepStr: '10'))
-        timeout(time: 1, unit: 'HOURS')
+        timeout(time: 30, unit: 'MINUTES')
         timestamps()
     }
 
+    environment {
+        LANG = 'en_US.UTF-8'
+        LC_CTYPE = 'en_US.UTF-8'
+        /* These environment variables make it feasible for Git to clone properly while
+         * inside the wacky confines of a Docker container
+         */
+        GIT_COMMITTER_EMAIL = 'me@hatescake.com'
+        GIT_COMMITTER_NAME = 'Hates'
+        GIT_AUTHOR_NAME = 'Cake'
+        GIT_AUTHOR_EMAIL = 'hates@cake.com'
```

Comment:
Should we really use this email "hates@cake.com" this domain exit, is used and we have no control on it? 
It would be better to use ci@jenkins.io (which doesn't exist) or jenkins@example.com
Same for me@hatescake.com